static/src/js/middleware/metrics/__tests__/events.test.js

```diff
@@ -385,6 +389,54 @@ describe('events', () => {
     })
   })
 
+  describe('add collection to project', () => {
+    test('pushes to the dataLayer', () => {
+      const action = {
+        type: METRICS_ADD_COLLECTION_PROJECT,
+        payload: {
+          collectionConceptId: 'C100000-EDSC',
+          page: 'page of origin',
```

Comment:
Instead of `page of origin`, use an actual value to better show how we are using these values. Same for `view`